sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/sources/TextSocketStreamSuite.scala

```diff
@@ -256,6 +246,66 @@ class TextSocketStreamSuite extends StreamTest with SharedSQLContext with Before
     }
   }
 
+  test("verify ServerThread only accepts the first connection") {
+    serverThread = new ServerThread()
+    serverThread.start()
+
+    withSQLConf("spark.sql.streaming.unsupportedOperationCheck" -> "false") {
```

Comment:
I would use `SQLConf.UNSUPPORTED_OPERATION_CHECK_ENABLED.key` BTW.

Reply:
Yeah actually I blindly copied the code line in the file. Agreed it would be better to use the key.

Reply:
Thanks for guiding, addressed.